sklearn/metrics/scorer.py

```diff
@@ -33,14 +34,15 @@
 
 
 class _BaseScorer(six.with_metaclass(ABCMeta, object)):
-    def __init__(self, score_func, sign, kwargs):
+    def __init__(self, score_func, sign, kwargs, deprecation_msg=None):
```

Comment:
I think that we will want to get rid of the "deprecation_msg" in two release. Maybe add a comment here saying that this should be removed for 0.20.